src/11_6_2021_RcppClock_benchmarking_Rcpp_code.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Benchmarking Rcpp code with RcppClock"
+author: "Zach DeBruine"
+license: GPL (>= 2)
+tags: Rcpp benchmarking RcppClock
+summary: Rcpp benchmarking with a simple C++ tick-tock clock and seamless integration
+  with the R environment at runtime.
+---
+
+Rcpp is all about the need for speed. However, Rcpp code is not magically fast, but is the result of careful coding and profiling. RcppClock makes it easy to profile C++ code from R:
+
+* On the C++ side, RcppClock wraps `std::chrono::high_resolution_clock` to time chunks of code, and exports the results to an R object in the global environment.

```

Comment:
Maybe note the implicit C++11 requirement which, as luck would have it, is a given in 2021 unless you work on a jurassic system.  If you ever had a job or internship you may have seen those.  (Hint: RHEL, CentOS, and others ...)

Reply:
C++11 requirement for `std::chrono::high_resolution_clock` noted.